community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/executionplan/StandardInternalExecutionResult.scala

```diff
@@ -122,12 +122,16 @@ abstract class StandardInternalExecutionResult(context: QueryContext, runtime: R
 
       override protected def createInner: util.Iterator[util.Map[String, Any]] = result.iterator()
 
-      override def executionPlanDescription(): InternalPlanDescription =
-        self.executionPlanDescription()
-          .addArgument(Planner(planner.toTextOutput))
-          .addArgument(PlannerImpl(planner.name))
```

Comment:
What happens with the Planner argument? Is that set somewhere else? I think benchmarking relies on it being set.

Reply:
This change only applies to internal testing through `RewindableExecutionResult`.